src/app/submission/components/process-details/process-details.component.ts

```diff
@@ -199,6 +229,27 @@ export class ProcessDetailsComponent implements OnInit {
       }));
   }
 
+  private getInputFiles(processUrl: string): Observable<ListResult<MetadataDocument>> {
+    return this.ingestService.get<ListResult<MetadataDocument>>(`${processUrl}/inputFiles`).pipe(
+      tap(data => {
+        const inputs = data._embedded ? data._embedded.files : [];
+        inputs.map(input => {
+          this.nodes.push({
+            id: input.uuid.uuid,
+            label: input.content['file_core']['file_name'],
+          } as NgxNode);
```

Comment:
I would argue that making this into a function would increase readability.

```typescript
ngxNodeFromInput(input) : NgxNode {
    return {
        id: input.uuid.uuid,
        label: input.content['file_core']['file_name'],
    }
}
```

This way you could even have a very elegant function chain:

```typescript
inputs.map(ngxNodeFromInput).forEach(this.nodes.push)
```